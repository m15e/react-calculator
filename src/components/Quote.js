import React from 'react';
import Header from './Header';

const Quote = () => (
  <>
    <Header />
    <h1>Hi from Quote</h1>
  </>
);

export default Quote;
