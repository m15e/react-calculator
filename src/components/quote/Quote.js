import React from 'react';
import Header from '../header/Header';

const Quote = () => (
  <>
    <Header />
    <div className="content-container">
      <h2>Quote of the day</h2>
      <p>What if I fall? Oh, but my darling, what if you fly?</p>
      <span>- Winnie the Pooh</span>
    </div>
  </>
);

export default Quote;
