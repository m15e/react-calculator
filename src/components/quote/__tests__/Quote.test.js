import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../Quote';

it('Quote component matches snapshot', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
