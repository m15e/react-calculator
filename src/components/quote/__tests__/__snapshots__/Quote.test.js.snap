// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote component matches snapshot 1`] = `
Array [
  <div
    className="header-container"
  >
    <ul>
      <li>
        <a
          href="/"
        >
          Math Magicians
        </a>
      </li>
      <li>
        <a
          href="/"
        >
          Home
        </a>
      </li>
      <li>
        <a
          href="/calculator"
        >
          Calculator
        </a>
      </li>
      <li>
        <a
          href="/quote"
        >
          Quote
        </a>
      </li>
    </ul>
  </div>,
  <div
    className="content-container"
  >
    <h2>
      Quote of the day
    </h2>
    <p>
      What if I fall? Oh, but my darling, what if you fly?
    </p>
    <span>
      - Winnie the Pooh
    </span>
  </div>,
]
`;
